### ISSUE TITLE
Parallel Genomes Layout

### ISSUE THREAD

**Issue by josiahseaman** [josiahseaman/DDV#11]
**Motivation:** Visualize the same chromosome from 3 different species aligned together in one static visualization.  

Take 3 files as inputs and layout every column of the file in parallel with the 2 other files.  Create another gap level to group the sets of 3 columns together.

**Comment by josiahseaman** [comment]
![image](https://cloud.githubusercontent.com/assets/2256148/16393444/15582d78-3c6e-11e6-951d-37f6b3e50772.png)

**Comment by josiahseaman** [comment]
![image](https://cloud.githubusercontent.com/assets/2256148/16394749/00ff21fa-3c74-11e6-9bce-642ce21bfdfb.png)

These are totally unrelated files, so it will look pretty incoherent.

**Comment by josiahseaman** [comment]
Here's the first useful 3 column visualization.  This is the history of Human Chromosome 9 from most recent to oldest: v38, v37, v35.
![image](https://cloud.githubusercontent.com/assets/2256148/16396434/ec82611c-3c7b-11e6-83be-4bbc64909759.png)

You can see all three stay in sync on the upper chromosome arm, but they desync at the centromere.  v37 has sequence that extends the farthest into the centromere.  v38 alpha satellite sequence starts eariler than the others, etc.  Near the end of the file, it's still possible to visually align pieces from really bright elements, but overall they're pretty out of sync.

Below I highlight the same green double band in all 3 versions, but v37 is so much longer than the others, it ends up on the next row.
![image](https://cloud.githubusercontent.com/assets/2256148/16396607/c2fe4184-3c7c-11e6-80c0-4bc01f6ac270.png)

Clearly alignment will be necessary to make this mode usable.

**Comment by josiahseaman** [comment]
I've added the ability to render any number of files in parallel.  I found that they were easier to differentiate if there was a faint colored border around them.  Now, I'm going to add Colored Titles in the image, so people know which is which.  Colored borders will also have the nice advantage that gaps from alignment will be colored by their column's background color.  So it should be pretty easy to see relative amounts of gaps at a macro scale just from color.
